build.gradle

```diff
@@ -23,6 +23,7 @@ buildscript {
         springSecurityOauth2Version = '2.5.2.RELEASE'
         springSecurityOauth2AutoconfigureVersion = '2.6.8'
         mariadbJdbcVersion = '2.7.9' // Bumping to v3 breaks some pipeline jobs, so pinning to v2 for now. v2 (current version) is stable and will be supported until about September 2025 (https://mariadb.com/kb/en/about-mariadb-connector-j/).
+        snakeyamlVersion = '2.0'
```

Comment:
It looks like this is related to the "update snakeyaml" from the commit message.

Maybe "upgrade" would be more accurate? 

I don't understand why this is updating or upgrading the dependency. It looks like it was not pinned to anything before, which often means that we are always pulling the latest version. So yeah, I don't understand what is happening.

Reply:
There are CVEs for snakeyaml that the default version of spring boot uses, this updates the usage of snakeyaml to be be an overridden version 2.0 which does not have any vulnerabilities. This is the update that has been applied in uaa 76 and 74 and is now being applied in credhub.

Reply:
@bruce-ricard  let me know if this is clear or if additional explanation is needed.

Reply:
I'll take thumbs up as this is clear. closing and merging, thank you!